### PR TITLE
add tiger80 iso layout support

### PR DESCRIPTION
**Pull request by moyi4681**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/tiger80/config.h [hunk 1]
```diff
@@ -13,28 +13,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
```
Comment (zvecr) [inline keyboards/kbdfans/tiger80/config.h:15]:
```suggestion
 */
 
 #pragma once
```

File: keyboards/kbdfans/tiger80/keymaps/iso/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+enum layers {
+    _LAYER0,
+    _LAYER1,
+    _LAYER2,
+    _LAYER3,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [_LAYER0] = LAYOUT_tkl_f13_iso_tsangan(
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/iso/keymap.c:28]:
```suggestion
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {

    [0] = LAYOUT_tkl_f13_iso_tsangan(
```

File: keyboards/kbdfans/tiger80/info.json [hunk 3]
```diff
@@ -1,119 +1,230 @@
...
+        "saturation_steps": 10,
+        "sleep": true
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0011",
+        "vid": "0x4B42"        
     },
     "community_layouts": ["tkl_f13_ansi_tsangan"],
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/info.json:43]:
```suggestion
    "community_layouts": ["tkl_f13_ansi_tsangan", "tkl_f13_iso_tsangan"],
```

File: keyboards/kbdfans/tiger80/info.json [hunk 4]
```diff
@@ -1,119 +1,230 @@
...
-    "url": "", 
-    "maintainer": "kbdfans", 
-    "usb": {
-        "vid": "0x4B42",
-        "pid": "0x0011",
-        "device_version": "0.0.1"
+    "keyboard_name": "Tiger80",
+    "maintainer": "kbdfans",
+    "bootloader": "atmel-dfu",
+    "debounce": 5,
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/info.json:6]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/tiger80/rules.mk [hunk 5]
```diff
@@ -1,12 +0,0 @@
...
-#
-BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/rules.mk:12]: If this file is empty, it should contain only
```make
# This file intentionally left blank
```

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**